src/Microsoft.Sbom.Common/Config/ManifestToolActions.cs

```diff
@@ -12,5 +12,6 @@ public enum ManifestToolActions
     Validate = 1,
     Generate = 2,
     Redact = 4,
-    All = Validate | Generate | Redact
+    ValidateFormat = 8,
```

Comment:
Why 8 here?

Reply:
It looks like the enum was constructed as a bitmask, so our next binary flag would be 8.

This made me curious to go look and see why we even conceive of this enum as a bitmask - since in practice you could not OR together different actions to run them. Turns out we use it as an applicability filter on custom attributes that can be applied to the different command line flags.